Client/src/Validation/error.js

```diff
@@ -22,15 +22,29 @@ const hasValidationErrors = (form, validation, setErrors) => {
 					"dbConnectionString",
 					"refreshTokenTTL",
 					"jwtTTL",
+					"notify-email-list",
 				].includes(err.path[0])
 			) {
 				newErrors[err.path[0]] = err.message ?? "Validation error";
 			}
+			// Handle conditionally usage number required cases
+			if (!form.cpu || (form.cpu && form.usage_cpu)) {
+				delete newErrors["usage_cpu"];
+			}
+			if (!form.memory || (form.memory && form.usage_memory)) {
+				delete newErrors["usage_memory"];
+			}
+			if (!form.disk || (form.disk && form.usage_disk)) {
+				delete newErrors["usage_disk"];
+			}
```

Comment:
What are these checks? Not sure why the condition to deleting those keys

Reply:
Side note as well in general it's best to set the value of the key to undefined or null rather than deleting the key value pair.

Reply:
these checks are for the conditional error setting for the threshold inputs, the inputs are be mandatory when the checkbox are checked;

Reply:
> Side note as well in general it's best to set the value of the key to undefined or null rather than deleting the key value pair.

yes, Have changed to set them as null instead

Reply:
> What are these checks? Not sure why the condition to deleting those keys

these checks are for the conditional error setting for the threshold inputs, the inputs are mandatory when the checkbox are checked;

Reply:
resolved in : https://github.com/bluewave-labs/bluewave-uptime/pull/1149/commits/a9aece6123eaa20ce2173c62c7f7904379a9f0dd
https://github.com/bluewave-labs/bluewave-uptime/pull/1149/commits/4e60981b0b1c487182b830991c89c117f6670006 etc